audio/src/main/java/dev/masonroot/audio/exceptions/VoskInitializationException.java

```diff
@@ -0,0 +1,39 @@
+package dev.masonroot.audio.exceptions;
+
+import java.nio.file.Path;
+
+/**
+ * Exception thrown when the Vosk STT engine fails to initialize with the specified model path.
+ *
+ * <p>This exception is used to indicate issues during the initialization of the Vosk speech
+ * recognition engine, such as when the model path is invalid or the model cannot be loaded.
+ *
+ * <p><b>Notes:</b>
+ *
+ * <ul>
+ *   <li>This exception extends {@code SttEngineInitialization} to provide more specific error
+ *       information related to Vosk initialization.
+ * </ul>
+ */
+public final class VoskInitializationException extends EngineInitializationException {
```

Comment:
This exception should have a constructor which accepts the cause so that the stack trace contains the most amount of relevant information for debugging. Currently information is lost when this exception is being thrown as seen in VoskTtEngine on line 58 where the causing IOException is erased.